cmd/cored/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # cored changelog
 
+## 1.1.0 (February 24, 2017)
+
+This release is a minor version update, and contains **new features** and **deprecations**. cored 1.1.0 is backward-compatible with 1.0.x SDKs, but we strongly recommend upgrading to 1.1.x SDKs as soon as possible. See SDK-level changes here:
+
+- [Java SDK changelog](https://github.com/chain/chain/blob/main/sdk/java/CHANGELOG.md)
+- [Node SDK changelog](https://github.com/chain/chain/blob/main/sdk/node/CHANGELOG.md)
+- [Ruby SDK changelog](https://github.com/chain/chain/blob/main/sdk/ruby/CHANGELOG.md)
+
+Other changes:
+
+* The network version has been updated to **2**. Chain Core instances on the same network must share the same network version. If you're upgrading to version 1.1.0, make sure to upgrade all Chain Cores in your blockchain network.
+* Transaction outputs now have a unique `id` property.
+* Transaction inputs refer to previous outputs using a new `spent_output_id` property. The existing `spent_output` property, which contains a transaction ID and position, is **deprecated**.
+* Accounts now use **receivers**, a cross-core payment primitive that supersedes the Chain 1.0.x pattern of creating and paying to control programs. See the SDK changelogs for usage examples.
+
```

Comment:
I think I might go into that level of detail in the SDK docs themselves, since it gets a bit involved.